README.md

```diff
@@ -37,15 +37,10 @@ git clone git@github.com:uzh-rpg/rpg_vid2e.git --recursive
 ```
 
 ## Installation with [Anaconda](https://www.anaconda.com/distribution/)
-Adapt the CUDA toolkit version according to your setup. 
 
 ```bash
-cuda_version=10.1
-
-conda create -y -n vid2e python=3.7
-conda activate vid2e
-conda install -y pytorch torchvision cudatoolkit=$cuda_version -c pytorch
-conda install -y -c conda-forge opencv tqdm scikit-video eigen boost boost-cpp pybind11 matplotlib
+conda create --name vid2e --file requirements.txt
```

Comment:
best to test that this works without problems. I often run into freezes with requirement/env files.